third-party/realdds/scripts/topic-send.py

```diff
@@ -0,0 +1,76 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+from argparse import ArgumentParser
+args = ArgumentParser()
+args.add_argument( '--debug', action='store_true', help='enable debug mode' )
+args.add_argument( '--quiet', action='store_true', help='No output; just the minimum FPS as a number' )
+args.add_argument( '--device', metavar='<path>', help='the topic root for the device' )
+args.add_argument( '--topic', metavar='<path>', help='the topic on which to send flexible message, if --device is not supplied' )
+args.add_argument( '--message', metavar='<string>', help='a message to send', default='some message' )
+def domain_arg(x):
+    t = int(x)
+    if t <= 0 or t > 232:
+        raise ValueError( f'--domain should be [0,232]' )
+    return t
+args.add_argument( '--domain', metavar='<0-232>', type=domain_arg, default=0, help='DDS domain to use (default=0)' )
+args = args.parse_args()
+
+
+if args.quiet:
+    def i( *a, **kw ):
+        pass
+else:
+    def i( *a, **kw ):
+        print( '-I-', *a, **kw )
+def e( *a, **kw ):
+    print( '-E-', *a, **kw )
+
+
+import pyrealdds as dds
+import time
+import sys
+
+dds.debug( args.debug )
+
+settings = {}
+
+participant = dds.participant()
+participant.init( args.domain, 'fps', settings )
```

Comment:
Update name and argument description. Not FPS related

Reply:
Will fix